benchmark/es/map-bench.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+const common = require('../common.js');
+const assert = require('assert');
+
+const bench = common.createBenchmark(main, {
+  method: ['object', 'nullProtoObject', 'fakeMap', 'map'],
+  millions: [10]
+});
+
+function runObject(n) {
+  const m = {};
+  var i = 0;
+  bench.start();
+  for (; i < n; i++) {
+    m['i' + n] = n;
```

Comment:
Are we trying to set the same two properties (`'i10000000'` and `'s10000000'`) 10 million times? If so, maybe use keys like `'a', 'b'`. Or should the keys be `'i' + i` and `'s' + i`?